monkeytype/encoding.py

```diff
@@ -2,7 +2,8 @@
 # All rights reserved.
 #
 # This source code is licensed under the BSD-style license found in the
-# LICENSE file in the root directory of this source tree.
+# LICENSE file in the root directory of this source tree. An additional grant
```

Comment:
Looks like this came from an older version of this file? Eliminate this addition.